Gemfile

```diff
@@ -46,3 +46,4 @@ group :test do
   gem 'webmock'
 end
 
+gem 'coverband'
```

Comment:
This gem added after all others, per the [reference app](https://github.com/coverband-service/coverband_demo/blob/master/Gemfile)